sciLifeLab_utils/run_QC_report.py

```diff
@@ -394,6 +407,8 @@ class MyTheme(DefaultTheme):
     parser.add_argument('--qc-folder', type=str, required=True,
             help=("Path to where denovo QC pipeline was run. Where you find, "
                 "the sample folders P????_??? eg. in, /foo/B.Baz_15_01/01-QC/"))
+    parser.add_argument('--delivery-folder', type=str, required=False,
+            help="Where the delivery will be staged")
```

Comment:
These arguments in `run_qc_analysis.py`, `run_assembly.py` and etc. are not really used. They were just left there for legacy reasons, but now that we have a documentation that says otherwise we can just remove them.

The real parameters are found in the file `run_denovo.py`. And you need to add this there, ie:
```@click.option('--delivery-folder', ...)```